stable/python/_kubeless.py

```diff
@@ -97,18 +97,63 @@ def handler():
                 return res
 
 
+def preload():
+    """This is a no-op function used to start the forkserver."""
+    pass
+
+
 if __name__ == '__main__':
-    import sys
+    import logging
+    import multiprocessing as mp
     import requestlogger
+
+    mp_context = os.getenv('MP_CONTEXT', 'forkserver')
```

Comment:
Using `forkserver` by default because it is faster than `spawn` and also because it allows us to preload data that can be used by all requests as shown in `stable/python/examples/preload.py`.

If `forkserver` is not available, we get back a `ValueError` from `mp.get_context()` in which case we will default to `spawn`.